src/components/TodoItem.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useCallback } from "react";
+import styled, { css } from "styled-components";
+import { ITodoItemProps } from "interface";
+
+const TodoItem = ({
+  type,
+  todo,
+  idx,
+  deleteCurrentList,
+  addToggleList,
+}: ITodoItemProps) => {
```

Comment:
```suggestion
const TodoItem = ({
  type,
  todo,
  idx,
  deleteCurrentList,
  addToggleList,
}: ITodoItemProps): React.FC<ITodoItemProps> => {
```

보통은 이런 식으로 직접 컴포넌트를 타이핑해준답니다.

Reply:
아하 넵 저걸 빼먹었네요!